src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -57,64 +62,75 @@ class RequestorStep extends React.Component {
             isOnfidoSetupComplete: lodashGet(props, ['achData', 'isOnfidoSetupComplete'], false),
         };
 
+        // Required fields not validated by `validateIdentity`
         this.requiredFields = [
             'firstName',
             'lastName',
-            'requestorAddressStreet',
-            'requestorAddressCity',
-            'requestorAddressZipCode',
-            'dob',
-            'ssnLast4',
-            'requestorAddressState',
+            'isControllingOfficer',
         ];
+
+        // Map a field to the key of the error's translation
+        this.errorTranslationKeys = {
+            firstName: 'bankAccount.error.firstName',
+            lastName: 'bankAccount.error.lastName',
+            isControllingOfficer: 'requestorStep.isControllingOfficerError',
+        };
+
+        this.clearError = inputKey => clearError(this.props, inputKey);
+        this.getErrors = () => getErrors(this.props);
     }
 
-    onFieldChange(field, value) {
+    getErrors() {
+        return lodashGet(this.props, ['reimbursementAccount', 'errors'], {});
+    }
+
+    /**
+     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
+     *
+     * @param {String} inputKey
+     * @param {String} value
+     */
+    clearErrorAndSetValue(inputKey, value) {
         const renamedFields = {
             street: 'requestorAddressStreet',
             city: 'requestorAddressCity',
             state: 'requestorAddressState',
             zipCode: 'requestorAddressZipCode',
         };
-        const fieldName = lodashGet(renamedFields, field, field);
-        const newState = {[fieldName]: value};
+        const renamedInputKey = lodashGet(renamedFields, inputKey, inputKey);
+        const newState = {[renamedInputKey]: value};
         this.setState(newState);
         updateReimbursementAccountDraft(newState);
+
+        if (inputKey === 'dob') {
+            this.clearError('dobAge');
+        }
```

Comment:
NAB, I think this logic could benefit from some brief documentation explaining that `dob` is unique from other fields as there are two kinds of validation happening. 

Probably eventually we could have something like `dob_invalid` and `dob_minimumAge` where the front part indicates the `fieldName` and the next part indicates which error. Or move to a structure like

```
errors: {
    identity: {
        dob: {
            invalid: true,
            minimumAge: true,
        }
    }
}
```

Reply:
I'm leaving the same comment on each part we do something with 'dob' related to multiple errors/validations

Reply:
I like this structure 😬 

```
errors: {
    identity: {
        dob: {
            invalid: true,
            minimumAge: true,
        }
    }
}
```

Reply:
+1 this looks like a good way of handling the multiple errors